openstates/in/bills.py

```diff
@@ -77,9 +81,127 @@ def get_name(self,random_json):
         #got sick of doing this everywhere
         return random_json["firstName"] + " " + random_json["lastName"]
 
+    def process_votes(self,rollcalls,bill,proxy):
+
+        result_types = {
+            'FAILED': False,
+            'DEFEATED': False,
+            'PREVAILED': True,
+            'PASSED': True,
+            'SUSTAINED': True,
+            'NOT SECONDED': False,
+            'OVERRIDDEN': True,
```

Comment:
Could you explain this one a bit more in a comment? Are we talking about veto override? Objection override? How do we know that this is always going to be a positive for the bill?